content/enterprise_influxdb/v1.8/guides/_index.md

```diff
@@ -8,8 +9,8 @@ menu:
     weight: 60
 ---
 ## [Anti-entropy: Replace Data Node](/enterprise_influxdb/v1.8/administration/anti-entropy/)
-## [Backing up and restoring in InfluxDB Enterprise](/enterprise_influxdb/v1.8/administration/backup-and-restore/)
+## [Back up and restore in InfluxDB Enterprise](/enterprise_influxdb/v1.8/administration/backup-and-restore/)
 ## [Fine-grained authorization in InfluxDB Enterprise](/enterprise_influxdb/v1.8/guides/fine-grained-authorization/)
-## [Migrating InfluxDB OSS instances to InfluxDB Enterprise clusters](/enterprise_influxdb/v1.8/guides/migration/)
-## [Rebalancing InfluxDB Enterprise clusters](/enterprise_influxdb/v1.8/guides/rebalance/)
+## [Migrate InfluxDB OSS instances to InfluxDB Enterprise clusters](/enterprise_influxdb/v1.8/guides/migration/)
+## [Rebalance InfluxDB Enterprise clusters](/enterprise_influxdb/v1.8/guides/rebalance/)
 ## [SMTP server setup](/enterprise_influxdb/v1.8/guides/smtp-server/)
```

Comment:
Probably best to replace these with the `children` shortcode, but we can come back and do it later.